core/templates/dev/head/pages/exploration-player-page/learner-experience/conversation-skin.directive.ts

```diff
@@ -825,6 +834,13 @@ oppia.directive('conversationSkin', [
                 return;
               }
             }
+            if (ENABLE_SOLICIT_ANSWER_DETAILS_FEATURE) {
```

Comment:
I'm not sure whether I understood the workflow correctly but I see a cyclic loop here, can you explain in brief (maybe using flowdiagram)?

Reply:
Actually when the learner will submit the answer info for an answer then that state will get registered in the visited state names. So if the list is in visited state names, then no cycle will be there. Hope it helps, if not feel free to ask.